### ISSUE TITLE
Movim 0.12

### ISSUE THREAD

**Issue by matlag** [movim/movim_ynh#38]
Hello!

Movim 0.12 has just been released!
Can we have the package update?

Thanks!

**Comment by src386** [comment]
Hello matlag,

Yes it's possible, it is now commited :) You can upgrade movim_ynh. The installer now uses movim HEAD. This means every time you upgrade movim_ynh, you will have the latest available movim.

**Comment by matlag** [comment]
Works just fine!
Thanks very much!

**Comment by matlag** [comment]
Hi there,

I just (shame on me) figured that we're actually still not on the right branch.
The update goes fine with "HEAD", but "HEAD" does not seem to point to the right version.

I'm not expert in git (I'm actually not a dev guy), but based on what I read, in order for "HEAD" to point to the "master" branch, we need to set it so, otherwise "HEAD" just points to the latest version pointed (so we udpate on the same version over and over).

[git-remote man page](https://www.kernel.org/pub/software/scm/git/docs/git-remote.html)

For what I understand from this, we need to issue a command somewhere such as
`git-remote set-head origin master`
in order for "HEAD" to point to "master".

Or maybe I'm completely misunderstanding how that works.
Bottom line is I'm reopening this ticket...

**Comment by src386** [comment]
Thank you for reporting this issue. I'm currently checking.

**Comment by JimboJoe** [comment]
As a general rule of thumb for a YunoHost package, you shouldn't use the upstream head in the upgrade script. We want to make sure the used version in fully compatible with YunoHost, so every upgrade of the app reference (commit id or release) should be tested against current stable YunoHost version (hence XMPP server in this case).
Does that make sense?

**Comment by src386** [comment]
@JimboJoe there is no real review or QA procedure for movim_ynh so I thought a fixed movim version was not  relevant.

BUT upgrading from a fixed commit to HEAD was more complicated than I expected, plus I had weird issues with the latest movim commit, I don't know if it's upstream or not. So yeah the fixed version system is back for now.

@matlag can you try to update your movim installation ?

    yunohost app upgrade movim -f https://github.com/movim/movim_ynh

**Comment by matlag** [comment]
@src386 It works!
"À propos" tells me it's Movim 0.12

**Comment by JimboJoe** [comment]
@src386 I understand your point, but having a fixed version is required in order to become official, and I still think your package should become official in the future as we don't have any official xmpp client any more since jappix deprecation 😉 

By the way, the upgrade just went fine for me 👍
